test/utils/Bytes.t.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity ^0.8.20;
+
+import {Test} from "forge-std/Test.sol";
+import {Bytes} from "@openzeppelin/contracts/utils/Bytes.sol";
+
+contract BytesTest is Test {
+    function testSymbolicEqual(bytes memory a, bytes memory b) public pure {
+        assertEq(Bytes.equal(a, b), Bytes.equal(a, b));
+    }
```

Comment:
is that what you had in mind ?
```suggestion
    function testSymbolicEqual(bytes memory a) public pure {
        assertTrue(Bytes.equal(a, a));
    }
```

Reply:
testing equality that way is nice, but we'll also need some inequality unit tests. Not fuzzing, just plain an simple tests (same length/different length). Because of the complexity of building bytes objects in solidity, I'd personally do it hardhat